core/src/epicli/data/common/ansible/playbooks/roles/haproxy_runc/files/extract-docker-image-V1.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
```

Comment:
I would use a docker daemon to control haproxy container lifecycle, but if there is such standard/requirement, this is ok.

Reply:
So my rationale for not using docker is that docker does not work well with systemd, because of the client-server architecture (here's some source of knowledge https://github.com/ibuildthecloud/systemd-docker on the topic). Instead in this PR I'm using runc which seems to be working fine with systemd. My rationale for not using just the docker itself (without systemd) is that we have never done that (as far as I know) and Epiphany seems to be systemd-exclusive. I think if there is a common approval for using just the docker I can live with that. Any more votes for it? 🤔